src/main/java/de/blablubbabc/insigns/Utils.java

```diff
@@ -63,7 +48,7 @@ public static DyeColor getSignTextColor(org.bukkit.block.Sign sign) {
 	public static void sendSignUpdate(Player player, org.bukkit.block.Sign sign) {
 		Preconditions.checkNotNull(player, "player is null");
 		Preconditions.checkNotNull(sign, "sign is null");
-		player.sendSignChange(sign.getLocation(), sign.getLines(), getSignTextColor(sign), sign.isGlowingText());
+		player.sendBlockChanges(List.of(sign));
```

Comment:
Are you sure that this works as expected? Looking at the code, I would assume that this only updates block data, not tile entity data.

However, what about using the simpler `sendBlockUpdate(Location location, TileState tileState)` instead?

Reply:
that's right, I accidentally used the wrong api method here